README.md

```diff
@@ -1,57 +1,33 @@
-![MIT License](https://img.shields.io/github/license/iacsecurity/tool-compare)
```

Comment:
This section wasn't supposed to be deleted :) granted, the README generator only generated part of the README. We could change the generator to include the rest of the README and not just the test results. I can handle that in separate item. WDYT @owenrumney ?

Reply:
added back in